src/back/auth/src/service/session.hpp

```diff
@@ -19,11 +19,25 @@ class Session final {
 	model::Session Create(
 		const OIDCTokens& tokens,
 		const TokenPayload& data,
-		const std::string& userAgent);
+		const std::string& userAgent,
+		const std::chrono::system_clock::time_point& exp);
+
+	model::Session Refresh(
+		const OIDCTokens& tokens,
+		const TokenPayload& data,
+		const std::string& userAgent,
+		const std::chrono::system_clock::time_point& exp,
+		const boost::uuids::uuid& oldSessionId);
 
 private:
 	table::Session& _table;
 	tokens::Session& _tokenizer;
+
+	model::Session Bootstrap(
+		const OIDCTokens& tokens,
+		const TokenPayload& data,
+		const std::string& userAgent,
+		const std::chrono::system_clock::time_point& exp);
```

Comment:
1. Слишком абстрактное имя с учётом того что в классе могут появится и другие методы которые не будут вызывать этот. Предлагаю prepare или prepareNew
2. Приватные методы предлагаю называть с маленькой буквы. Это позволяет отличить из в файле определения и при автодополнении в некоторых случаях.
```suggestion
	model::Session prepare(
		const OIDCTokens& tokens,
		const TokenPayload& data,
		const std::string& userAgent,
		const std::chrono::system_clock::time_point& exp);
```

Reply:
Поправил.